internal/bminventory/inventory_v2_handlers.go

```diff
@@ -643,3 +643,37 @@ func (b *bareMetalInventory) infraEnvIPXEScript(ctx context.Context, infraEnv *c
 
 	return fmt.Sprintf(ipxeScriptFormat, kernelURL, rootfsURL, initrdURL), nil
 }
+
+func (b *bareMetalInventory) GetInfraEnvIPXEScriptDownloadURL(ctx context.Context, params installer.GetInfraEnvIPXEScriptDownloadURLParams) middleware.Responder {
```

Comment:
Besides getting the URL is it going to be stored? 
I don't remember this part and i didn't find it in the enhancement

Reply:
> it going to be stored?

No, only in the infraenv kube-api.